tests/lit-tests/2468.ispc

```diff
@@ -0,0 +1,23 @@
+// RUN: %{ispc} %s --target=avx2-i32x8 --arch=x86-64 --nostdlib --emit-asm -o - | FileCheck %s
+
+// REQUIRES: X86_ENABLED
+
+// The goal of this test is to check that code generation results into a pair of loads, an add, and a store
+
+// CHECK-COUNT-1: vmovups
```

Comment:
`CHECK-COUNT-1` is equivalent to `CHECK`, as far as I understand. I think it's not checking that "no more than expected number of matches happen".